wordpress-mobile/gutenberg-mobile
# Move BlockToolbar to Gutenberg

Step 5 of https://github.com/wordpress-mobile/gutenberg-mobile/issues/958
This is the next step in our port of the BlockList to gutenberg.

Gutenberg PR https://github.com/WordPress/gutenberg/pull/16213

### Testing Instructions
- Boot the app on iOS and Android
- Check that the block toolbar (toolbar that appears at the bottom of the app) is still functional meaning you can add a block, undo and redo, but also edit formats in a paragraph block etc.
